catalog/video/video-mpegts-stream/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -77,7 +77,14 @@
             <property name="stanag4609Processor">
                 <bean class="org.codice.alliance.libs.klv.Stanag4609ProcessorImpl">
                     <argument>
-                        <bean class="org.codice.alliance.libs.klv.OffsetCenterPostProcessor"/>
+                        <bean class="org.codice.alliance.libs.klv.ListPostProcessor">
+                            <argument>
+                                <list>
+                                    <bean class="org.codice.alliance.libs.klv.TrimmingPostProcessor"/>
+                                    <bean class="org.codice.alliance.libs.klv.OffsetCenterPostProcessor"/>
```

Comment:
Why not do a reference list of `PostProcessor` here instead of manually instantiating a bean?